scala/init.sh

```diff
@@ -11,10 +11,15 @@ SCALA_VERSION="2.10.3"
 
 if [[ "0.7.3 0.8.0 0.8.1" =~ $SPARK_VERSION ]]; then
   SCALA_VERSION="2.9.3"
+  wget http://s3.amazonaws.com/spark-related-packages/scala-$SCALA_VERSION.tgz
+elif [[ "2.0.0" =~ $SPARK_VERSION ]]; then
+  SCALA_VERSION="2.11.8"
+  wget http://downloads.lightbend.com/scala/2.11.8/scala-$SCALA_VERSION.tgz
```

Comment:
I've also uploaded this to the s3 bucket. Lets switch to that to avoid depending on the lightbend source ?